app/src/main/java/com/glowstudio/android/blindsjn/ui/navigation/MainNavGraph.kt

```
@@ -0,0 +1,195 @@
+    ) {
+        // 홈 화면
+        composable("home_screen") {
+            topBarViewModel.updateState(TopBarState("홈 화면", false, false))
+            HomeScreen(navController = navController)
+        }
+
+        composable("news_detail/{articleJson}") { backStackEntry ->
+            val articleJson = backStackEntry.arguments?.getString("articleJson")
+            val article = try {
+                Gson().fromJson(URLDecoder.decode(articleJson, "UTF-8"), Article::class.java)
+            } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)